### ISSUE TITLE
UnknownPropertyException when opening the default avatar elements browsers

### ISSUE THREAD

**Issue by arifje** [verbb/comments#299]
### Describe the bug

When trying to select an default avatar the elements browsers doesn't show anything and is causing an exception;

```
#12 {main} {"memory":33410664,"exception":"[object] (yii\\base\\UnknownPropertyException(code: 0): Setting unknown property: craft\\elements\\db\\AssetQuery::enabledForSite at /var/www/site/cms/vendor/yiisoft/yii2/base/Component.php:209)"} 
```

Caused by: **enabledForSite: null** (in templates-default.twig settings template). 

When I remove this parameter it works again.

### Craft CMS version

4.8.5

### Plugin version

2.0.11

### Multi-site?

no

**Comment by engram-design** [comment]
Fixed for the next release. To get this early run `composer require verbb/comments:"dev-craft-4 as 2.0.11"`

**Comment by engram-design** [comment]
Fixed in [2.0.12](https://github.com/verbb/comments/releases/tag/2.0.12)
